newlib/libm/test/conv_vec.c

```diff
@@ -3692,5 +3713,7 @@ double_type doubles[] =
 {__LINE__, "0x1.0p-969", 0x1.0p-969, 10 },
 {__LINE__, "0x1.0p-970", 0x1.0p-970, 10 },
 #endif
+#else
```

Comment:
Don't think these elses at the end of the arrays are required with the new structure.

Reply:
thanks, I'd almost forgotten to fix that. Done.